extension/content.js

```diff
@@ -282,11 +299,18 @@ $(document).on('click', event => {
 	$target.closest('.js-details-container').toggleClass('refined-github-minimized');
 });
 
+// Handle issue list ajax
+$(document).on('pjax:end', event => {
+	addYoursMenuItem();
```

Comment:
```js
if (pageDetect.isIssueSearch() || pageDetect.isPRSearch()) {
    addYoursMenuItem('pulls');
}
```

Also, you might want to add a test that the button doesn't exist before adding it. Some pages don't do a full refresh of the DOM and you could have the button appended multiple times. (It might be premature optimization, but we had to do this pattern elsewhere.)

Adding something like this will make sure we don't add a button if it already exists:

```js
if ($(`.subnav-item[href="/${type}?q=is%3Aopen+is%3Aissue+user%3A${username}"]`).length > 0) {
    return;
}
```